pkg/handlers/v1/dependencycheck.go

```diff
@@ -0,0 +1,24 @@
+package v1
+
+import (
+	"context"
+
+	"github.com/asecurityteam/ipam-facade/pkg/domain"
+)
+
+// DependencyCheckHandler takes in a domain.DependencyChecker to check external dependencies
+type DependencyCheckHandler struct {
+	DependencyCheckList []domain.DependencyCheck
```

Comment:
This is an abstraction leak. The handler shouldn't need to know whether the `DependencyCheck` interface implementors performs a single check or multiple, so that if that changes over time, the handler code will not need to be changed.

Task: Wrap all individual dependency checks in a `DependencyChecker` interface implementation that is configured with N `DependencyChecker` implementations, and whose own implementation fails immediately on the first failure. Bind this to the handle such that is only needs to return the value from a single `CheckDependencies` invocation.

Reply:
done